subsys/bluetooth/host/gatt.c

```diff
@@ -1094,6 +1094,7 @@ ssize_t bt_gatt_attr_write_ccc(struct bt_conn *conn,
 		}
 
 		bt_addr_le_copy(&cfg->peer, &conn->le.dst);
+		cfg->id = conn->id;
```

Comment:
We might need to update ccc_find_cfg since that apparently don't match the id.

Reply:
Possibly. This will require more changes as `ccc_save` does not carry id.

Reply:
It is encoded in the key, check bt_gatt_store_ccc

Reply:
I will try to add changes needed. Will update PR when done.

Reply:
Should be done